src/transformers/commands/add_new_model_like.py

```diff
@@ -1189,6 +1189,13 @@ def create_new_model_like(
             if "tokenization" not in str(f) and "processor" not in str(f) and "feature_extraction" not in str(f)
         ]
 
+    def disable_fx_test(filename: Path):
+        with open(filename) as fp:
+            content = fp.read()
+        with open(filename, "w") as fp:
+            new_content = re.sub(r"fx_ready\s*=\s*True", "fx_ready = False", content)
```

Comment:
Nit, this line should go before the second `with`.

Reply:
Done!